### PR TITLE
feat:archive functionality

### PR DESCRIPTION
**Pull request by Aazam10**
1.Archive /Unarchive functionality
2.Add to trash from archive
Note : Notes validation has not been added so for now user can add empty notes
Please login with test credentials before testing functionality.
Commented console.logs for now will remove them in later version

### CODE REVIEW

**Review by gadgetfather — APPROVED** [review]
- well-written code 
- One thing I would suggest is to redirect the user to the login page when he/she is trying to add notes without logging in.
- Also when I am deleting notes from the archive it is going into the trash but when I restore them from the trash it is going back to the normal notes page. i think it should go to archived. you can work on that.

Great work @Aazam10

File: src/Services/ArchiveServices/addToArchive.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+const addToArchive = (id, token, note) => {
```
Comment (Rohanmond) [inline src/Services/ArchiveServices/addToArchive.js:2]: better to write async here to define that this function is returning a promise

File: src/Services/ArchiveServices/addToTrashFromArchive.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const addToTrashFromArchive = (id, token, note) => {
```
Comment (Rohanmond) [inline src/Services/ArchiveServices/addToTrashFromArchive.js:3]: here also

**Review by Rohanmond — APPROVED** [review]
Folder structure looks good. Ready to merge

File: src/Services/ArchiveServices/addToTrashFromArchive.js (hunk 2 above)
Comment (Aazam10) [reply src/Services/ArchiveServices/addToTrashFromArchive.js:3]: Thanks will do it in next pr

**Pull request merged**